pkg/csi_driver/controller.go

```diff
@@ -40,6 +40,9 @@ const (
 	enterpriseTier = "enterprise"
 	defaultNetwork = "default"
 
+	defaultConnectMode = "DIRECT_PEERING"
+	privateConnectMode = "PRIVATE_SERVICE_ACCESS"
```

Comment:
privateServiceAccess

Reply:
wouldn't the `defaultConnectMode` be more aligned with the `defaultNetwork` above